a3/terraform/modules/cluster/gke-beta/scripts/gke_node_pool.sh

```diff
@@ -31,23 +31,29 @@ gke_node_pool::create () {
 
     echo "Creating node pool '${node_pool_name}' in cluster '${cluster_name}'..." >&2
     gcloud beta container node-pools create "${node_pool_name}" \
+        --image-type="CUSTOM_CONTAINERD" \
+        --image=cos-105-17412-156-30 \
+        --image-project=cos-cloud \
         --cluster="${cluster_name}" \
         --region="${region}" \
+        --node-locations="${zone}" \
+        --project="${project_id}" \
+        --machine-type="${machine_type}" \
+        --num-nodes="${node_count}" \
+        --disk-type="${disk_type}" \
+        --disk-size="${disk_size}" \
```

Comment:
I think we dont necessarily need these after specifying the --ephemeral-storage-local-ssd flag.

Reply:
Are we sure about removing these or should we still keep these are optional (so that we don't have to make a code change if we need this in future) ?

Reply:
i'm actually not sure if this is compatible with the ephemeral storage local ssd option. I personally prefer using the bundled ssd since remote ssd quota is very tight and if users choose to create a cluster with remote ssds they will very likely need to recreate their node pools when scaling things up.